src/policy/policy.cpp

```diff
@@ -46,6 +46,8 @@ bool IsStandard(const CScript& scriptPubKey, txnouttype& whichType, const bool w
             return false;
         if (m < 1 || m > n)
             return false;
+    } else if (whichType == TX_HTLC) {
+        return false;
```

Comment:
It looks like you're assuming all TX_HTLC transactions are DOS attacks?
Why reject them all?